tests/functional/s3/test_cp_command.py

```diff
@@ -211,19 +211,6 @@ def test_no_metadata_directive_for_non_copy(self):
         self.assertEqual(self.operations_called[0][0].name, 'PutObject')
         self.assertNotIn('MetadataDirective', self.operations_called[0][1])
 
-    def test_cp_succeeds_with_mimetype_errors(self):
```

Comment:
I moved it to this test: https://github.com/aws/aws-cli/pull/2117/files#diff-25df2ee263c482c355f21087bad3b373R250

It seemed like a really poor functional test: we were patching mimetypes in a functional test that really should have been in a unit test. If you want to keep the test, I am not opposed to adding it back, but I was just trying to clean some code up.